### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by yorickdowne**
- Typo in README
- Restart in docker-compose.yml
- `exec geth` so it can receive `SIGTERM`
- `ROLLUP_POLL_INTERVAL_FLAG` throughout so l2geth uses it

### CODE REVIEW

File: env.example [hunk 1]
```diff
@@ -1,7 +1,8 @@
+DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=https://kovan-ec.example.com
```
Comment (optimisticben) [inline env.example:1]: This should be removed, yeah?

**Review by optimisticben — CHANGES_REQUESTED** [review]
small change, the rest looks good!

File: env.example (hunk 1 above)
Comment (yorickdowne) [reply env.example:1]: Hmm I don't think so. Without `DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT` the replica won't work, so it needs to be in `.env`.

Comment (optimisticben) [reply env.example:1]: Oh sure, an endpoint is needed. I meant the value. We can leave 
`DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=`
or something like
`DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=WONT_WORK_UNLESS_YOU_PROVIDE_A_VALID_L1_ENDPOINT`
I was leaving it out and hoping people would read the docs.

Comment (yorickdowne) [reply env.example:1]: Ah right. How about with `DATA_TRANSPORT_LAYER__L1_RPC_ENDPOINT=https://AN-ETHEREUM-RPC-ENDPOINT-HERE-OR-IT-WONT-WORK`

**Comment by yorickdowne** [comment]
Since the change to the structure went in first, will fix this to match

**Pull request merged**